research/src/pages/contribute.mdx

```diff
@@ -206,9 +206,27 @@ pathToProposal: /components/test
 ---
 ```
 
-There are some MDX components available in Open UI's docs to help analyze and understand the data collected.
-You can reference some other component doc pages for usage examples.
-Otherwise, you can collect and analyze your data more manually.
+### Research page content
+
+After this front matter, copy in the content from the [Component Research Template](https://github.com/WICG/open-ui/blob/master/research/src/pages/component-research-template.mdx).
+
+This template includes a few optional partials you can import into the research page.
+Each of these walks through the design system JSON5 files in `/sources` to finds components where the `openUIName` value matches the `Name` value of your research page front matter.
+
+- **Anatomy:** lists out component anatomy by part,
+  if any matching components have defined an anatomy.
+- **Concept Coverage:** lists matching components according to unique names,
```

Comment:
Not sure that the name "concept coverage" is very clear for this one. Maybe this should be a subsection under concepts, "also known as"?

Reply:
Thanks for the review Una! I think that's out of the scope for this PR since I'm just documenting what's there w/r/t partials, but I like the idea of renaming this partial. Maybe worth opening a new issue?

Reply:
SGTM